### PR TITLE
Fix readme

### PR DESCRIPTION
**Pull request by kevinadda**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,78 +62,92 @@ genericLogger.indexDocument(document, function(error, body) {
...
-    id: "recipientId"
-  },
-  message: {
-    text: "hello, world!"
-  }
-}
-```
+Log the message inside the POST request callback made when the bot replies (the user message and bot response(s) must be logged in the same document) (see right pane).
+
+<a type="button" target="_blank" class="button" href="https://github.com/Botfuel/sample-bot-messenger">
```
Comment (windkomo) [inline README.md]: This looks a bit strange on github I think 🤣

File: README.md [hunk 2]
```diff
@@ -62,78 +62,92 @@ genericLogger.indexDocument(document, function(error, body) {
...
 });
 ```
 
+
 ### With Bot builder
 
-We provide you with a middleware you can easily plug to Microsoft bot builder:
+We provide a middleware that you can easily plug to Microsoft bot builder (see right pane).
+
+<a type="button" target="_blank" class="button" href="https://github.com/Botfuel/sample-bot-botbuilder"><i style="padding-right: 6px;" class="fa fa-github fa-lg"></i> See our sample bot</a>
```
Comment (windkomo) [inline README.md]: Same

File: README.md [hunk 3]
```diff
@@ -62,78 +62,92 @@ genericLogger.indexDocument(document, function(error, body) {
...
         }
     })
 });
 ```
 
+
 ### With Bot builder
 
-We provide you with a middleware you can easily plug to Microsoft bot builder:
+We provide a middleware that you can easily plug to Microsoft bot builder (see right pane).
```
Comment (windkomo) [inline README.md]: No right pane here

File: README.md (hunk 1 above)
Comment (kevinadda) [reply README.md]: You think? 😆 
image

Comment (kevinadda) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (kevinadda) [reply README.md]: removed

File: README.md (hunk 2 above)
Comment (kevinadda) [reply README.md]: Removed

**Pull request merged**